### DISCUSSION TITLE
Indicazioni su casi d'uso Matr_005,Matr_013

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#729] [Supporto Webapp] [answered]
Con la presente chiediamo un chiarimento circa i casi in cui ci si attende si faccia riferimento ai casi d'uso in oggetto.

**Matr_005, Matr_013**
Per matrimonio celebrato fuori dalla casa comunale, dovrebbe intendersi il matrimonio, preceduto da pubblicazioni, che per motivi di	 necessità dimostrata viene celebrato in un luogo appartenente al territorio comunale ma	esterno alla "Casa Comunale" (art.	110	c.c.). Considerato che per quest'ultimo caso esiste in effetti "Matr_002", il nostro dubbio è che con 
Matr_005
Matr_013
ci si intenda riferire ai matrimoni celebrati al di fuori della sede municipale. Questa possibilità è ammessa dalla norma, purché si tratti di luoghi di cui il comune abbia disponibilità individuati tramite apposita delibera di Giunta; questi, per effetto della delibera medesima, divengono però a tutti gli effetti "casa comunale".

In particolare, ancora:
- per Matr_005 si indica come riferimento la formula 120 che fa evidente riferimento nel testo ad un matrimonio che si celebra a seguito di presentazione degli sposi all'usc e non contiene alcun riferimento ad eventuale impedimento dello sposo ovvero della sposa
- per Matr_013 si indica come riferimento la formula 120 che fa evidente riferimento nel testo ad un matrimonio che si celebra a seguito di presentazione degli sposi all'usc e non contiene alcun riferimento ad eventuale impedimento dello sposo ovvero della sposa

Rimaniamo in attesa di vostra indicazione in merito

**Comment by asqueglia** [comment]
Buongiorno @antonioluberto essendo una casistica per la quale non sono espressamente censite le formule al riguardo, sulla web app abbiamo composto la minuta adattando la formula 120 usata per il matrimonio su delega altro comune introducendo la parte presente sulla formula 118 usata per i matrimoni fuori della casa comunale, per trattare appunto l'impedimento. Approfondiremo meglio con ANUSCA le casistiche per chiarire se sono fattispecie che possono verificarsi, nel qual caso di chiarire la formula da usare,  oppure se sono casi d'uso da eliminare.

**Comment by asqueglia** [comment, marked as answer]
Buongiorno @antonioluberto in merito alla richiesta abbiamo fatto un approfondimento con Anusca. ANUSCA conferma che sebbene sia una casistica poco frequente può comunque capitare per cui si confermano i casi d'uso e si conferma l’adattamento fatto della formula. Chiudiamo pertanto la discussione. In caso ulteriori domande si prega di aprirne una altra dedicata
